nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2918,7 +3022,7 @@ The Provenance Repository contains the information related to Data Provenance. T
 
 |====
 |*Property*|*Description*
-|nifi.provenance.repository.implementation|The Provenance Repository implementation. The default value is `org.apache.nifi.provenance.PersistentProvenanceRepository`.
+|`nifi.provenance.repository.implementation`|The Provenance Repository implementation. The default value is `org.apache.nifi.provenance.PersistentProvenanceRepository`.
 Two additional repositories are available as well.
```

Comment:
This should be "Three additional repository implementations are available as well."